packages/webpack-cli/lib/bootstrap.js

```diff
@@ -23,6 +24,11 @@ const runCLI = async (cliArgs) => {
         // If the unknown arg starts with a '-', it will be considered an unknown flag rather than an entry
         let entry;
 
+        // enable/disable colors
+        if (coloretteOptions.enabled && !parsedArgs.opts.color) {
+            coloretteOptions.enabled = parsedArgs.opts.color;
+        }
```

Comment:
We should respect here `--colors` and `--no-colors`

Reply:
I think we are or am I missing something?

Reply:
So if user provide the value we should respect it, if not colorette automatically enable/disable it
```js
if (typeof parsedArgs.opts.color !== 'undefined') {
  coloretteOptions.enabled = Boolean(parsedArgs.opts.color);
}
```

Reply:
Actully we have `color: true` by default. So `parsedArgs.opts.color` is always true. 

https://github.com/webpack/webpack-cli/blob/46cba367f06a6354fe98fcb15e7771e819feeac0/packages/webpack-cli/lib/utils/cli-flags.js#L95

But we can add this check to make it more future proof.

Reply:
We should remove this default value, because it is invalid colors autodetecting inside `colorretter`